### PR TITLE
Get and Delete Activities by ID

### PR DESCRIPTION
**Pull request by psdupvi**
Added one endpoint and renamed another. Also updated .gitignore

- Renamed `get_activity_evaluation` to `get_activity` and updated docstring/logger, because this returns all activity summary, not just the evaluation. Also updated example.py
- Added `delete_activity`, which accepts an activity id, and deletes that activity from Garmin Connect
- Updated the .gitignore file to add `ignore/` and `.idea/`. Adding `ignore/` allows for easy local testing in a confined folder without having to worry about accidentally committing extra files. `.idea/` is an automatically generated PyCharm folder, if you use PyCharm

### CODE REVIEW

**Comment by psdupvi** [comment]
Issue https://github.com/cyberjunky/python-garminconnect/issues/187

File: garminconnect/__init__.py [hunk 1]
```diff
@@ -1022,13 +1039,13 @@ def get_activity_hr_in_timezones(self, activity_id):
 
         return self.connectapi(url)
 
-    def get_activity_evaluation(self, activity_id):
-        """Return activity self evaluation details."""
+    def get_activity(self, activity_id):
```
Comment (psdupvi) [inline garminconnect/__init__.py:1042]: Possibly it makes sense to leave get_activity_evaluation as is, and create another (identical) method called get_activity, but to me, that feels less good.  

Maybe it makes sense to add a warning if someone has been calling get_activity_evaluation -- leave it as a function but have it call get_activity instead? I'm open to suggestions, but I figured I'd create the PR to get the ball rolling

**Comment by psdupvi** [comment]
Sorry, accidentally closed PR while cleaning up my own branches

**Comment by cyberjunky** [comment]
Thanks! Going for the rename option and mention it as a breaking change!

**Pull request merged**